### ISSUE TITLE
Update ml5 p5 version to v0.10.2

### ISSUE THREAD

**Issue by joeyklee** [ml5js/ml5-examples#265]
Dear ml5 community, 

I'm submitting a new issue. Please see the details below.

### → Step 1: Describe the issue 📝
> Did you find a bug? Want to suggest an idea for a new example?

* maintenance
* update p5 version to latest -  v0.10.2 // https://p5js.org/get-started/

**Comment by joeyklee** [comment]
@bomanimc - Just making a note that we may also consider adding to our devOps processes a nice way to update all the p5 references as versions change. The easy route is just a global `find` + `replace` but we should at the very least document this.

**Comment by bomanimc** [comment]
Good idea! This might also be a good place for some of the import templating proposed in https://github.com/ml5js/ml5-library/issues/809.
